terraform/environments/staging/dns.tf

```diff
@@ -79,50 +57,6 @@ resource "google_dns_record_set" "discourse" {
   ttl     = 300
 }
 
-# VPN-protected DNS records
-
-resource "google_dns_record_set" "metabase" {
```

Comment:
@AndrewDryga, did you need this for any reason?

Reply:
This was for `metabase.firez.one` which was the old BI tracker that has since been deprovisioned in Vultr.